DetectorDescription/DDCMS/interface/DDSolidShapes.h

```diff
@@ -67,21 +78,23 @@ namespace cms {
     ddcons = 4,
     ddpolycone = 5,
     ddpolyhedra = 6,
-    ddtorus = 7,
-    ddunion = 8,
-    ddsubtraction = 9,
-    ddintersection = 10,
-    ddshapeless = 11,
-    ddpseudotrap = 12,
-    ddtrunctubs = 13,
-    ddsphere = 14,
-    ddellipticaltube = 15,
-    ddcuttubs = 16,
-    ddextrudedpolygon = 17,
-    ddtrd1 = 18,
+    ddunsupported1 = 7,
```

Comment:
I do not think, the enum should be modified since there is a mapping between legacy shapes and the new shapes:
```
Old id: 0 == new id: 0
Old id: 1 == new id: 1
Old id: 2 == new id: 2
Old id: 3 == new id: 3
Old id: 4 == new id: 4
Old id: 5 == new id: 5
Old id: 7 == new id: 5
Old id: 6 == new id: 6
Old id: 8 == new id: 6
Old id: 9 == new id: 7
Old id: 10 == new id: 8
Old id: 11 == new id: 9
Old id: 12 == new id: 10
Old id: 13 == new id: 11
Old id: 14 == new id: 12
Old id: 15 == new id: 13
Old id: 16 == new id: 14
Old id: 17 == new id: 15
Old id: 18 == new id: 16
Old id: 19 == new id: 17

```